planetarium/NineChronicles
# Introduce local market service runner

### Description

1. Run local market service from local MarketService repository. 

### How to test

1. Clone [MarketService](https://github.com/planetarium/market-service) and install dependencies.
2. Select MarketService directory from clone 1 
<img width="836" alt="image" src="https://user-images.githubusercontent.com/3193043/230576868-809f5539-97d8-4379-904f-229ad156d6cf.png">
3. Configure `appsettings.local.json` for target rpc headless node.

```
{
  "NoMiner": true,
  "RpcClient": true,
  "Language": "en",
  "RpcServerHost":"9c-main-test-1.nine-chronicles.com",
  "RpcServerPort": 31238,
  "marketServiceHost": "http://127.0.0.1:5000",
  "genesisBlockPath": "https://release.nine-chronicles.com/genesis-block-9c-main"
}
```

<img width="258" alt="image" src="https://user-images.githubusercontent.com/3193043/230577445-33236d72-1a47-474f-a636-fe448268ec19.png">
5. check `UseLocalMarketService` option and play game. 


### Related Links

### Required Reviewers

### Screenshot
